cluster/module/server.c

```diff
@@ -1235,6 +1235,13 @@ write_status(gchar *path)
 	g_free(services_names);
 }
 
+static time_t
+get_file_timestamp(gchar *path)
+{
+	GStatBuf buf;
+	g_stat(path, &buf);
```

Comment:
Return code **MUST** be checked. Returning a field from an uninitiated structure is *a bad thing*. With a recent compiler this would raise a huge warning.